src/purchases.ts

```diff
@@ -732,17 +801,26 @@ export default class Purchases {
   }
 
   /**
-   * @deprecated, use enableAdServicesAttributionTokenCollection instead.
-   * Enable automatic collection of Apple Search Ad attribution. Disabled by default
-   * @param {boolean} enabled Enable or not automatic apple search ads attribution collection
-   * @returns {Promise<void>} The promise will be rejected if configure has not been called yet.
+   * iOS only. Always returns an error on iOS < 15.
+   *
+   * Use this method only if you already have your own IAP implementation using StoreKit 2 and want to use
+   * RevenueCat's backend. If you are using StoreKit 1 for your implementation, you do not need this method.
+   *
+   * You only need to use this method with *new* purchases. Subscription updates are observed automatically.
+   *
+   * Important: This should only be used if you have set PurchasesAreCompletedBy to MY_APP during SDK configuration.
+   *
+   * @warning You need to finish the transaction yourself after calling this method.
+   *
+   * @param {string} productID Product ID that was just purchased
+   * @returns {Promise<PurchasesStoreTransaction>} If there was a transacton found and handled for the provided product ID.
```

Comment:
We dont call out macOS specifically here, but I don't think we do it in the iOS SDK either?

Reply:
Yeah, we don't call it out in the iOS SDK. At the very least, the macOS limitation/use case for this is called out in the migration guide!